src/app/app.lastSearchesService.js

```diff
@@ -0,0 +1,29 @@
+export default class LastSearchesService {
```

Comment:
Same as before, `app.` prefix is not necessary (`lastSearch.service.js`, for example). In this case, a folder called services can be interesting :)